FitNesseRoot/DbFit/AcceptanceTests/JavaTests/TeradataTests/FlowMode/SimpleStoredProc/content.txt

```diff
@@ -1,6 +1,6 @@
 This stored proc tests output parameters - CalcLength returns string length as second output param
 
-|Execute Ddl|!-Replace Procedure CalcLength(in name varchar(100), out strlength numeric)
+|Execute Ddl|!-Replace Procedure CalcLength(in name varchar(100), out strlength bigint)
```

Comment:
We're still covered in `StoredProcWithException`.

None of the data type tests for Teradata stored price and functions are explicit enough really. I wonder if we could define some standard tests sections and use `!include`s to supply the specific content for each DB sometime.